docs/how-to/develop-customise/build-kernel.md

```diff
@@ -139,10 +139,11 @@ environment and the necessary scripts have execute permissions:
 
 ```{code-block} shell
 cd <kernel_source_working_directory>
-chmod a+x debian/rules
-chmod a+x debian/scripts/*
-chmod a+x debian/scripts/misc/*
-fakeroot debian/rules clean
+```
+```{code-block} shell
+chmod a+x debian/scripts/* && \
```

Comment:
@khbecker could you take a look at this change?

@gagath , sorry, why don't we need the chmod for debian/rules?